verification/curator-service/api/src/controllers/uploads.ts

```diff
@@ -115,6 +115,7 @@ export default class UploadsController {
                             upload: '$uploads',
                         },
                     },
+                    { $sort: { 'upload.created': -1 } },
```

Comment:
This is probably cool if `created` has timestamp resolution, but we probably want an explicit secondary sort otherwise.

Reply:
you'll need an index as well for sorting, look at the schema for cases for an example (it should have a sort-related comment on it)

Reply:
I meant this:
https://github.com/globaldothealth/list/blob/main/data-serving/data-service/schemas/cases.indexes.json#L162

but note that it's not going to be an issue unless we reach a significant amount of uploads to go through so probably find to do in another PR (we started seeing OOMs due to sorting without an index at around 170K cases).

Reply:
Got it. I'll add the index in a separate PR since it will take some figuring out how to set up the first index for the curator-service (and I want to get the build fixes in).  

In terms of a secondary sort, I think it's okay to default to the order that was happening before we added this (I believe based on order added to the DB). WDYT?

Reply:
I'd say one of two things:

1. If `created` is a time (and not just a date), show the HH:MM and no need for a secondary sort.
2. If `created` is a date, like it's displayed, we should probably secondary sort by the source name. Just so it's easier to find the relevant upload and the table appears to be organized.

Reply:
Added secondary sort on source name